jest.config.js

```diff
@@ -3,33 +3,24 @@ const { relative } = require('path');
 
 const branch = execSync('git branch | grep "*" | cut -d " " -f2').toString();
 console.log(`Running tests on branch "${branch}"`);
-const base = branch === 'master' ? 'HEAD~1' : 'origin/canary';
-const diff = execSync(`git diff ${base} --name-only`).toString();
+const gitPath = branch === 'master' ? 'HEAD~1' : 'origin/canary...';
```

Comment:
```suggestion
const gitPath = branch === 'master' ? 'HEAD~1' : 'origin/canary...HEAD';
```

I think these are equivalent but let's be explicit.